# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/FileManager.h
# Commit: a09556b2333b
# Full Hash: a09556b2333bd856169ac0106f33a6df0bd821ac
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-04-22 04:10:25
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Eliminate output and raw pointer input parameters on some ActorsParent functions. r=dom-workers-and-storage-reviewers,ttung
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68338
# ==============================================================================

diff -r 9a6271146478 -r a09556b2333b dom/indexedDB/FileManager.h
--- a/dom/indexedDB/FileManager.h	Tue Apr 21 14:33:06 2020 +0000
+++ b/dom/indexedDB/FileManager.h	Tue Apr 21 14:40:00 2020 +0000
@@ -45,7 +45,7 @@
   static MOZ_MUST_USE nsCOMPtr<nsIFile> GetCheckedFileForId(nsIFile* aDirectory,
                                                             int64_t aId);
 
-  static nsresult InitDirectory(nsIFile* aDirectory, nsIFile* aDatabaseFile,
+  static nsresult InitDirectory(nsIFile& aDirectory, nsIFile& aDatabaseFile,
                                 const nsACString& aOrigin,
                                 uint32_t aTelemetryId);
 
